docs/user/customization.md

```diff
@@ -70,6 +70,25 @@ push down, editor.action.moveLinesDownAction
 
 Now when you say eg "push down air and bat", cursorless will first select the two tokens with a gray hat over the `a` and `b`, then issue the VSCode command `editor.action.moveLinesDownAction`, and then restore your original selection.
 
+## \[Experimental\] Cursorless custom regex scopes
+
+You can define custom scopes in `user/cursorless-settings/experimental/regex_scope_types.csv`
+
+> [!TIP]
+> Use `visualize [scope]` to see a live preview of the regex scope (must save `regex_scope_types.csv` to see changes)
+
+---
+Command `visualize dotted` with the following scope type `dotted,"[\w.]+"`
+
+![image](https://github.com/cursorless-dev/cursorless/assets/13824365/b54263a3-74b7-40d5-a1aa-cd7bbe86430b)
+
+You can then use commands such as
+* `take dotted sun` to select "user.text" (line 10)
+* `take dotted urge` to select "user.chrome_mod" (line 11)
+
+> [!NOTE]
```

Comment:
I might also make this a tip?

Reply:
Not clear what you wanted. Let me know after I send the new version if anything still needs changing.